### DISCUSSION TITLE
New Releases..

### DISCUSSION THREAD

**Discussion by WilsonBradley** [curif/AgeOfJoy-2022.1#99] [General]
Currently up to  v0.2.0 Release candidate 2

@curif @fabioECuri  thanks for releasing update..

I freaked out when the black boy did a cartwheel in front of me.. LOL.. Don't know if that's new or not.

Having less machines in each room seems to improve performance.
I won't be spending much time testing, until I can actually use as a Right handed person. (control section)

**Comment by curif** [comment]
The boy playing in the in the intro gallery is very old, LOL

The performance improvement It's not just because there are fewer machines, but also because of a new way to load cabinets. There are the same number of cabinets in the first room (001), but it feels more fluid.

**Reply by WilsonBradley** [reply]
Ya, but he never did a cartwheel in front of me at start.. Thank God you made them passthrough 
 ;-)
